feconf.py

```diff
@@ -807,3 +807,7 @@ def get_empty_ratings():
 
 VIEW_METHOD_ROLE = 'role'
 VIEW_METHOD_USERNAME = 'username'
+
+# Types of display in creatod dashboard
+CREATOR_DASHBOARD_DISPLAY_PREF_CARD = 'card'
```

Comment:
I think we should put these into assets/constants.js (which is shared between the frontend and backend), and refer to the constants in this file from the frontend, in order to avoid skew. We shouldn't be hardcoding strings in the frontend.

Reply:
Done